Main.java

```diff
@@ -9,7 +11,30 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //Ask the user for the number they want divided
+    System.out.println("Please enter your first number (This is the number you want divided):");
+    //initialize the users first number
+    int userNumberOne = input.nextInt();
+
+    //Ask the user for the number they want to divide by
+    System.out.println("Please enter your second number (This is the number you want to divide by):");
+    //initialize the users second number
+    int userNumberTwo = input.nextInt();
+    
+    //calculates the divided number and remainder
+    final int FINALNUMBER = userNumberOne / userNumberTwo;
+    final int FINALREMAINDER = userNumberOne % userNumberTwo;
+
+    //initializes both the equations for the final number and remainder
+    int numberFinal = FINALNUMBER;
+    int remainderFinal =FINALREMAINDER;
```

Comment:
You really don't need to use a constant here and the math could be done right in your variable on line 32. Otherwise, this code looks great! Just as an FYI as well, you can use numbers inside of variable names, they just can't start with one. For instance userNumberOne could be userNumber1